dashboard/app/views/layouts/application.html.haml

```diff
@@ -29,7 +30,10 @@
     -# auth key needed for statsig analytics
     %script{data: {'statsig-api-client-key': CDO.safe_statsig_api_client_key}}
     -# for statsig to recognized managed test server
-    %script{data: {'managed-test-server': "#{CDO.running_web_application? && CDO.test_system?}"}}
+    %script{data: {'managed-test-server': managed_test_server}}
+    -# user id and type for analytics logging
```

Comment:
nit: add a comment that this may not work for cached pages

Reply:
Added!